projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectCreationService.java

```diff
@@ -89,7 +108,7 @@ private Project createProject(String code, String title,
     ProjectCode projectCode;
     try {
       projectCode = ProjectCode.parse(code);
-      if(!projectRepository.find(projectCode).isEmpty()) {
+      if(projectRepository.find(projectCode).isEmpty()) {
```

Comment:
Please:
```suggestion
     projectRepository.find(...).ifPresentOrElse((), ())
```

Reply:
sadly does not work as we need to return in the block. with this syntax we cannot return. any return would return the lambda only

Reply:
😢